### ISSUE TITLE
Percent encoded output from `rake new_release`?

### ISSUE THREAD

**Issue by mokagio** [wordpress-mobile/release-toolkit#559]
While working on 11.0.0, I got this on my terminal:

```
>>> Opening PR draft in your default browser...
open https://github.com/wordpress-mobile/release-toolkit/compare/trunk...release/11.0.0?expand=1&title=Release%2011.0.0%20into%20trunk&body=Releasing%20new%20version%2011.0.0.

#%20What's%20Next

PR%20Author:%20Be%20sure%20to%20create%20and%20publish%20a%20GitHub%20Release%20pointing%20to%20`trunk`%20once%20this%20PR%20gets%20merged,
copy/pasting%20the%20following%20text%20as%20the%20GitHub%20Release's%20description:
```
["###%20Breaking%20Changes\n",%20"\n",%20"-%20Make%20`ios_check_beta_deps`%20use%20the%20`Podfile.lock`%20instead%20of%20`Podfile`%20for%20its%20detection,%20and%20also%20be%20able%20to%20detect%20Pods%20referenced%20by%20commits%20and%20branches.%20%20\n",%20"%20%20%20If%20your%20`Fastfile`%20called%20this%20action%20with%20an%20explicit%20`podfile:%20…`%20argument,%20you'll%20have%20to%20update%20the%20call%20to%20use%20`lockfile:`%20instead%20(or%20rely%20on%20defaults).%20[#557]\n",%20"\n"]
```

---------------

>>> WHAT'S NEXT

Once the PR is merged, publish a GitHub release for `11.0.0`, targeting `trunk`,
with the following text as description:

```
["### Breaking Changes\n", "\n", "- Make `ios_check_beta_deps` use the `Podfile.lock` instead of `Podfile` for its detection, and also be able to detect Pods referenced by commits and branches.  \n", "   If your `Fastfile` called this action with an explicit `podfile: …` argument, you'll have to update the call to use `lockfile:` instead (or rely on defaults). [#557]\n", "\n"]
```

The publication of the new GitHub release will create a git tag, which in turn will trigger
a CI workflow that will take care of doing the `gem push` of the new version to RubyGems.
```

This might have to do with my local setup, or maybe with some changes in Ruby or the tooling we use since the last release. I haven't looked into it yet, in favor of shipping the version.

**Comment by AliSoftware** [comment]
Probably related to recent change in the release script from https://github.com/wordpress-mobile/release-toolkit/pull/552

**Comment by AliSoftware** [comment]
Should be fixed by 7608326f4e0e449af8eb592ba6ac1ede304f01a1
